### PR TITLE
Refactor Utils a bit

### PR DESCRIPTION
**Pull request by giftymalik**
Get rid of extra variable and nested if condition

### CODE REVIEW

File: app/library/Utils.php [hunk 1]
```diff
@@ -25,14 +25,13 @@ class Utils extends Component
...
         if (true === is_object($data) && true === isset($data->$element)) {
-            $return = $data->$element;
+            return $data->$element;
         } elseif (true === is_array($data) && true === isset($data[$element])) {
-            $return = $data[$element];
+            return $data[$element];
         }
 
-        return $return;
+        return $default;
```
Comment (niden) [inline app/library/Utils.php]: Actually this was all done on purpose. There is the school of thought that you should always have one exit point or `return` statement in a method and this is what is shown here.

Indeed issuing a `return` exactly where it is needed (as you refactored it) is the fastest way possible for the function but in much larger methods all the `return` statements will cause maintenance headaches.

File: app/library/Utils.php [hunk 2]
```diff
@@ -45,10 +44,8 @@ public function getDocsUrl($lang)
         $return    = 'en';
         $languages = $this->config->get('doc_languages')->toArray();
 
-        if (true !== empty($lang)) {
-            if (true === array_key_exists($lang, $languages)) {
-                $return = $lang;
-            }
+        if (true !== empty($lang) && true === array_key_exists($lang, $languages)) {
```
Comment (niden) [inline app/library/Utils.php]: Put this in 2 lines to keep the 80 char limit per line please :)

File: app/library/Utils.php (hunk 1 above)
Comment (niden) [inline app/library/Utils.php]: Come to think of it, I did not mention this in the blog post. I should add this in the comment of the method

File: app/library/Utils.php (hunk 2 above)
Comment (giftymalik) [inline app/library/Utils.php]: Sure, BTW the CONTRIBUTING.md needs an update. I was not able to figure the exact branch to make this pull request to. If you could update it, would be a big help to report issues as well.

Comment (niden) [inline app/library/Utils.php]: @giftymalik Thank you! I KNEW I forgot something haha

**Comment by niden** [comment]
Thank you again!

**Pull request merged**